src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -35,7 +42,7 @@ private void PreProcessMonoDocXml()
 
             using (Progress.Start("Preprocessing monodoc XML files"))
             {
-                lock (s_lock)
+                lock (s_ecma)
```

Comment:
reuse `s_lock`?
I guess the lock is to assure that the repo content is locked for writing.
And `ecma` and `maml` are executed successively.

Reply:
I thought one is treating `xml` files, one is treating `maml` files, there should be no intersection and can be run in parallel